healthz/handlers/healthz.go

```diff
@@ -0,0 +1,25 @@
+package handlers
+
+import (
+	"net/http"
+	"io"
+	"github.com/gravitational/satellite/healthz/clusterstatus"
+	"github.com/gravitational/satellite/healthz/config"
+)
+
+func HandlerHealthz(w http.ResponseWriter, req *http.Request) {
+	config := req.Context().Value(config.ContextKey).(config.Config)
```

Comment:
Potential panic if the value is not of the expected type - you should either use type switch or additional boolean return value when type casting:
```go
config, ok := req.Context().Value(config.ContextKey).(config.Config)
if ok { // the type is correct
   // use config
}
```